docs/reference/changelog.md

```diff
@@ -18,7 +18,8 @@ sidebar_label: Changelog
 - Correct documentation of how code is formatted via terminal
   ([#294](https://github.com/darklang/docs/pull/294))
 
-- Fix some links to code ([#293](https://github.com/darklang/docs/pull/293))
+- Fix some links to Darklang code
```

Comment:
I was torn on this. "Darklang code" Feels like it implies code written in Darklang a bit. Probably fine though

Reply:
Yeah, this is a good point.